pipelines/agg_matched.yaml

```diff
@@ -36,6 +36,42 @@ tasks:
       connections.package: validate_drp
       connections.metric: AM3
       connections.agg_name: summary
+  agg_mean_AD1_design:
```

Comment:
I think we want "summary" measurements here.  If you really want the aggregation to be `mean`, you can see an example of how to override the default of `median` on line 9 of this file.

If we don't have "summary" measurements, they won't end up in the report or in SQuaSH.  We can call it something else, but that's what I did for the others.

Reply:
I think I'd also like the task name to say summary for consistency with the other definitions.  It's also no longer the case that this is computing a mean.

Reply:
Oops, yeah, I missed that. Fixed now.